actors/builtin/power/power_actor.go

```diff
@@ -304,6 +303,13 @@ func (a Actor) CurrentTotalPower(rt Runtime, _ *adt.EmptyValue) *CurrentTotalPow
 	rt.ValidateImmediateCallerAcceptAny()
 	var st State
 	rt.State().Readonly(&st)
+	if st.NumMinersMeetingMinPower < ConsensusMinerMinMiners {
+		return &CurrentTotalPowerReturn{
+			RawBytePower:     st.TotalRawBytePowerNoMin,
+			QualityAdjPower:  st.TotalQualityAdjPowerNoMin,
+			PledgeCollateral: st.TotalPledgeCollateral,
+		}
+	}
```

Comment:
The node's consensus system needs this same logic. Pull this out into an exported free function of state.